apps/emqx_resource/src/emqx_resource_app.erl

```diff
@@ -23,9 +23,18 @@
 -export([start/2, stop/1]).
 
 start(_StartType, _StartArgs) ->
+    %% since the handler is generic and executed in the process
+    %% emitting the event, we need to install only a single handler
+    %% for the whole app.
+    TelemetryHandlerID = telemetry_handler_id(),
```

Comment:
If I were a user, and I have multiple resources.
If my counters and gauges are all mixed together, it's maybe not quite helpful.